source/component/PasDoc_GenHtml.pas

```diff
@@ -2520,33 +2521,49 @@ function THTMLDocGenerator.MakeBodyBegin: string;
     i: Integer;
   begin
     Result := '';
+    Skip := false;
 
-    if Title <> '' then
+    if Introduction <> nil then
```

Comment:
I guess here applies the same which was mentioned in https://github.com/pasdoc/pasdoc/pull/95#pullrequestreview-417402790 and thus the `Skip` variable and additional code could be removed.

Reply:
I second that :) Just link to `index.html` always, and `Skip` should not be needed.

Reply:
ok